types/googlemaps/index.d.ts

```diff
@@ -304,7 +304,9 @@ declare namespace google.maps {
             stop(): void;
         }
 
-        // TODO find source documentation
-        class MapsEventListener {}
+        // https://developers.google.com/maps/documentation/javascript/reference/event#MapsEventListener
+        class MapsEventListener {
+            remove(): void;
```

Comment:
The [remove](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/googlemaps/reference/event.d.ts#L71) method has long been present in `google.maps.MapsEventListener`.

Reply:
`class MapsEventListener` remains as a rudiment and will be removed in the future.

Reply:
Ah of course, i did think it was a bit odd that this was missing.
Appologies. I didn't realise there were duplicate type definitions of the MapsEventListener class and had managed to find my self using this one.

Thanks for your help. I'll close this.